### PR TITLE
Handle dark mode selected by OS preference

### PR DESCRIPTION
**Pull request by krollins-mdb**
## What, How & Why?

The current TypeDoc theme doesn't properly account for an override when the dark theme is selected by OS preference. This PR fixes that.

### CODE REVIEW

**Review by kneth — APPROVED** [review]
Without any CSS skills: LGTM

**Comment by krollins-mdb** [comment]
Thank you, @kneth! Looks like I don't have write access to the Realm.js repo. Could you merge this PR for me?

**Comment by kneth** [comment]
@krollins-mdb And the tests are failing as you are not allowed to start the testing server

**Pull request merged**
